### PR TITLE
Use AJAX navigation for `default-branch-button`

### LINKED ISSUE #4623: `default-branch-button` does not use AJAX navigation
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Description

Clicking the `default-branch-button` should be the same as using the branch dropdown (except that we point to `/user/repo` and they open `/user/repo/tree/master`. Our link is better)

### Page navigation, wrong

Look at the bottom of the video, it says "page navigated"

https://user-images.githubusercontent.com/1402241/127604658-cbd90503-fbc3-4e59-b49e-9e39a8ff1141.mov

### Ajax navigation, correct

https://user-images.githubusercontent.com/1402241/127604642-a475b334-3ae7-4f5f-896b-e1d7b1aa4c29.mov

### Console errors

_No response_

### Example URL

https://github.com/vutran/twas/tree/v2.1.2

### Browser(s) used

Safari

### PR DESCRIPTION
**Pull request by dertieran**
Fixes #4623 

I just took the attribute from the `master` element and it worked 😅
It is also used for other elements so it seems to be the right way to trigger an AJAX navigation.

## Test URLs

https://github.com/vutran/twas/tree/v2.1.2

## Screenshot

https://user-images.githubusercontent.com/9264728/127857124-caf84808-6975-4b04-88bc-a82fe3439c1e.mp4

### Browser(s) used

Chromium 92.0.4515.107

### CODE REVIEW

**Review by yakov116 — APPROVED** [review]
🎉

**Comment by fregante** [comment]
Supper-lative! Are there any other Refined elements that would benefit from this attribute? I think most buttons we add on the repo could have that (like `latest-tag-button`). It might be worth looking for them

**Comment by dertieran** [comment]
Okay so I found out you can also just add a *(truthy)* `data-pjax` attribute and it would just make an AJAX.
*(I think it just used the `#js-repo-pjax-container` in that case, maybe because of the `data-pjax-container` attribute)*

But I'm not sure if I find the time to scan the code to add it to all elements.
If somebody want to add them feel free to do so, or I can also add them if I know the features and what container to use.
Otherwise I will see if I find the time later to look into it :)

**Comment by fregante** [comment]
Thanks for the PRs Philipp!

File: source/features/latest-tag-button.tsx [hunk 1]
```diff
@@ -132,6 +136,7 @@ async function init(): Promise<false | void> {
 				<a
 					className="btn btn-sm btn-outline tooltipped tooltipped-ne"
 					href={buildRepoURL(`compare/${latestTag}...${defaultBranch}`)}
+					data-pjax="#repo-content-pjax-container"
```
[the remark this answers is not in the document]
Comment (dertieran) [reply source/features/latest-tag-button.tsx:139]: True I totally overlooked that the header is duplicated 😅

**Pull request merged**